lib/src/components/NxTextLink/__tests__/NxTextLink.test.tsx

```diff
@@ -4,115 +4,96 @@
-    expect(getShallow()).not.toHaveProp('role', 'link');
-    expect(getShallow({ disabled: true })).toHaveProp('role', 'link');
+    expect(renderEl()).not.toHaveAttribute('role');
+    expect(quickRender({ disabled: true }).getByRole('link')).toBeInTheDocument();
   });
 
   it('does not pass href value on the text link when the disabled prop is set', function() {
-    expect(getShallow({ href: '#/pages/Text%20Link' })).toHaveProp('href', '#/pages/Text%20Link');
-    expect(getShallow({ href: '#/pages/Text%20Link', disabled: true })).toHaveProp('href', undefined);
+    expect(renderEl({ href: '#/pages/Text%20Link' })?.getAttribute('href')).toContain('#/pages/Text%20Link');
+    expect(renderEl({ href: '#/pages/Text%20Link', disabled: true })?.getAttribute('href')).toBe(null);
   });
 
   it('sets the target to _blank if newTab is true, unless a different target is specified', function() {
-    expect(getShallow()).toHaveProp('target', '');
-    expect(getShallow({ newTab: true })).toHaveProp('target', '_blank');
+    expect(renderEl()).toHaveAttribute('target', '');
+    expect(renderEl({ newTab: true })).toHaveAttribute('target', '_blank');
 
-    expect(getShallow({ target: 'asdf' })).toHaveProp('target', 'asdf');
-    expect(getShallow({ target: 'asdf', newTab: true })).toHaveProp('target', 'asdf');
+    expect(renderEl({ target: 'asdf' })).toHaveAttribute('target', 'asdf');
+    expect(renderEl({ target: 'asdf', newTab: true })).toHaveAttribute('target', 'asdf');
   });
 
   it('sets the target to _blank if the external prop is true and newTab is not false, unless a different target is ' +
       'specified', function() {
-    expect(getShallow({ external: true })).toHaveProp('target', '_blank');
-
-    expect(getShallow({ external: true, target: 'asdf' })).toHaveProp('target', 'asdf');
+    expect(renderEl({ external: true })).toHaveAttribute('target', '_blank');
+    expect(renderEl({ external: true, target: 'asdf' })).toHaveAttribute('target', 'asdf');
 
-    expect(getShallow({ external: true, newTab: true })).toHaveProp('target', '_blank');
-    expect(getShallow({ external: true, newTab: false })).toHaveProp('target', '');
+    expect(renderEl({ external: true, newTab: true })).toHaveAttribute('target', '_blank');
+    expect(renderEl({ external: true, newTab: false })).toHaveAttribute('target', '');
 
-    expect(getShallow({ external: true, noReferrer: true, target: 'asdf' })).toHaveProp('target', 'asdf');
-  });
-
-  it('adds an icon after the children if external is true', function() {
-    expect(getShallow({ children: <span/> })).not.toContainMatchingElement(NxFontAwesomeIcon);
-    expect(getShallow({ children: <span/>, external: true })).toContainMatchingElement(NxFontAwesomeIcon);
+    expect(renderEl({ external: true, noReferrer: true, target: 'asdf' })).toHaveAttribute('target', 'asdf');
   });
 
   it('wraps non-element children in an element sibling to the icon', function() {
```

Comment:
This test is probably also an implementation detail.  The span wrapper is just there so the styling works correctly, which is tested in the visual tests.

Reply:
> This test is probably also an implementation detail. The span wrapper is just there so the styling works correctly, which is tested in the visual tests.

Removed here: f1666b672214f5311a9852cb1c13ecf1de9419dd